### DISCUSSION TITLE
can't use  trackpad pinch to zoom

### DISCUSSION THREAD

**Discussion by nputhuva-rm** [SafeExamBrowser/seb-mac#217] [Q&A]
Hi,
I have added the settings to enable page zoom and text zoom in the configuration file still not able to zoom using trackpad gestures. (able to zoom using Keyboard shortcuts- Command & +/-). is this expected or not? if not, what should i do in addition to use trackpad gestures to zoom in and zoom out?

![image](https://user-images.githubusercontent.com/104403591/173184190-4e3a0442-11e8-4118-81d1-ba01749e6929.png)

**Comment by danschlet** [comment]
As [release notes](https://safeexambrowser.org/macosx/mac_release_notes_en.html) state, pinch to zoom in SEB for macOS  3.1 only works when the modern WebView is used. Study release notes and our developer documentation to figure out [how to use the modern WebView with an assessment solution](https://safeexambrowser.org/developer/seb-config-key.html).
